src/style.css

```diff
@@ -1,3 +1,91 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
The universal selector '*' is used here, which impacts performance. It's better to set box-sizing for elements that require it.